src/App.js

```diff
@@ -1,19 +1,63 @@
 import "./App.css";
+import "./components/LoginForm.scss";
 import { Routes, Route } from "react-router-dom";
+import { useState, useEffect } from "react";
 import Home from "./pages/home/Home";
 import Navbar from "./components/Navbar";
 import Footer from "./components/Footer";
 import Article from "./pages/article/Article";
 import Comments from "./pages/comments/Comments";
+import LoginForm from "./components/LoginForm";
+import NotFound from "./pages/error/NotFound";
+import { useNavigate } from "react-router-dom";
+import { fetchArticles } from "./api";
 
 function App() {
+  const [user, setUser] = useState("kyron");
+  const [article, setArticle] = useState({});
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      fetchArticles().then((articleData) => {
+        setArticles(articleData);
+        setIsLoading(false);
+      });
+    }, 1000);
+  }, []);
+
+  const navigate = useNavigate();
+
+  const handleClick = (article) => {
+    setArticle(article);
```

Comment:
The navigate bit is fine, but this setting isn't quite right. 
You need to get the article by doing a query in the single article card. 
This will use the article_id from the params/url in order to do this. 

At the moment, if you navigate to a single article via the url, it doesnt work. i.e. `/articles/11`